integration/test_named_vectors.py

```diff
@@ -562,3 +560,85 @@ def test_duplicate_named_vectors(collection_factory: CollectionFactory) -> None:
             ],
         )
         assert "Vector config names must be unique. Found duplicates" in str(e)
+
+
+@pytest.mark.parametrize(
+    "target_vector",
+    [
+        ["first", "second"],
+        wvc.query.TargetVectors.sum(["first", "second"]),
+        wvc.query.TargetVectors.minimum(["first", "second"]),
+        wvc.query.TargetVectors.average(["first", "second"]),
+        wvc.query.TargetVectors.manual_weights({"first": 1.2, "second": 0.7}),
+        wvc.query.TargetVectors.relative_score({"first": 1.2, "second": 0.7}),
+    ],
+)
+def test_named_vector_multi_target(
+    collection_factory: CollectionFactory, target_vector: Union[List[str], _MultiTargetVectorJoin]
+) -> None:
+    dummy = collection_factory("dummy")
+    if dummy._connection._weaviate_version.is_lower_than(1, 26, 0):
+        pytest.skip("Named vectors are not supported in versions lower than 1.26.0")
+
+    collection = collection_factory(
+        properties=[],
+        vectorizer_config=[
+            wvc.config.Configure.NamedVectors.none("first"),
+            wvc.config.Configure.NamedVectors.none("second"),
+        ],
+    )
+
+    uuid1 = collection.data.insert({}, vector={"first": [1, 0, 0], "second": [0, 1, 0]})
+    uuid2 = collection.data.insert({}, vector={"first": [0, 1, 0], "second": [1, 0, 0]})
+
+    objs = collection.query.near_vector([1.0, 0.0, 0.0], target_vector=["first", "second"]).objects
```

Comment:
The `target_vector` parametrized argument isn't being used here

Reply:
thx! Fixed now - I would expect a linter to warn there